src/components/Icons.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
just curious: are we going to use this component to show the corresponding chain icon?